libs/sdk-core/src/binding.rs

```diff
@@ -51,6 +51,85 @@ use crate::{
     SignMessageResponse, StaticBackupRequest, StaticBackupResponse,
 };
 
+use lazy_static::lazy_static;
+use log::{
+    max_level, set_boxed_logger, set_max_level, warn, Log, Metadata, Record, STATIC_MAX_LEVEL,
+};
+use parking_lot::RwLock;
```

Comment:
What's the reason to use `parking_lot` here? Can we use `std::sync::RwLock` or `tokio::sync::RwLock`?